tests/tests.cc

```diff
@@ -1,7 +1,5 @@
-#include "test_headers/common.h"
-
-// Automatically generated includes for the files in test_headers
-#include ".include_test_headers.h"
+// Google Test
+#include "gtest/gtest.h"
 
 int bobMain(int argc, char **argv)
```

Comment:
you don't **generally** need this with gtest, you can just link in gtest_main.cc

Reply:
Ah TIL! I think we will need it though as we already have a definition of ``main()`` that would conflict with the one in ``gtest_main.cc``.

Reply:
oh yeah, I guess main() is defined in the common module